src/main/java/liquibase/ext/databricks/snapshot/jvm/ColumnSnapshotGeneratorDatabricks.java

```diff
@@ -43,4 +59,71 @@ protected DataType readDataType(CachedRow columnMetadataResultSet, Column column
         }
         return super.readDataType(columnMetadataResultSet, column, database);
     }
+
+    @Override
+    protected DatabaseObject snapshotObject(DatabaseObject example, DatabaseSnapshot snapshot) throws DatabaseException {
+        //This should work after fix on Databricks side
+        if (example instanceof Column) {
+            Column column = (Column) super.snapshotObject(example, snapshot);
+            //These two are used too often, avoiding them? otherwise there would be too much DB calls
+            if (!column.getRelation().getName().equalsIgnoreCase("databasechangelog")
```

Comment:
tracking table can have custom names. 
better to use `database.getDatabaseChangeLogTableName()` and  `database.getDatabaseChangeLogLockTableName()`